src/sdk/modules/smartSessionsValidator/decorators/grantPermission.ts

```diff
@@ -1,4 +1,4 @@
-import type { ActionData, PolicyData, Session } from "@rhinestone/module-sdk"
+import { findTrustedAttesters, getTrustAttestersAction, MOCK_ATTESTER_ADDRESS, type ActionData, type PolicyData, type Session } from "@rhinestone/module-sdk"
```

Comment:
Instead of importing MOCK_ATTESTER_ADDRESS directly from module-sdk can you export it from our constants file the same way that we do the others?

Reply:
ok done